f2/src/forms/HowDidItStartForm.js

```diff
@@ -2,17 +2,16 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { useLingui } from '@lingui/react'
 import { Trans } from '@lingui/macro'
-import { Form, Field } from 'react-final-form'
```

Comment:
Field from react-final-form does not display label and helpertext. It displays just a text-input